OrbitCore/OrbitModuleTest.cpp

```diff
@@ -3,20 +3,52 @@
 
 #include <utility>
 
+#include "OrbitModule.h"
 #include "Path.h"
 #include "Pdb.h"
+#include "SymbolHelper.h"
 
-TEST(OrbitModule, LoadPdb) {
-  const std::string executable_path = Path::GetExecutablePath();
-  // This is a simple executable that prints "Hello Earh!"
-  const std::string file_path = executable_path + "testdata/hello_world_elf";
+const std::string executable_directory =
```

Comment:
🤔 I have clang-format running on every save. This should be fine ?!

Reply:
Should be indented with 2 spaces. Not sure why this isn't done by clang-format?!

Reply:
Well, there is no real sense in fighting clang-format, right? The next time someone runs it will indent it by 4 again. 

I tried to find something in the official style guide, but too long statements are not specifically mentioned.

Reply:
TEST is a macro. That's probably why it is not indented as expected. I would also not fight clang-format. Just creates a headache every single time you want to update the file in the future.

Reply:
I'm still unclear why this happens, but I suggest we stick to clang-format's wisdom.